lib/ansible/modules/cloud/azure/azure_rm_manageddisk.py

```diff
@@ -75,41 +74,40 @@
             - source_resource_uri
     os_type:
         description:
-            - "Type of Operating System: C(linux) or C(windows)."
-            - "Used when I(create_option) is either C(copy) or C(import) and the source is an OS disk."
-            - "If omitted during creation, no value is set."
-            - "If omitted during an update, no change is made."
-            - "Once set, this value cannot be cleared."
+            - Type of Operating System.
+            - Used when I(create_option=copy) or I(create_option=import) and the source is an OS disk.
+            - If omitted during creation, no value is set.
+            - If omitted during an update, no change is made.
+            - Once set, this value cannot be cleared.
         choices:
             - linux
             - windows
     disk_size_gb:
         description:
-            - "Size in GB of the managed disk to be created."
-            - "If I(create_option) is C(copy) then the value must be greater than or equal to the source's size."
+            - Size in GB of the managed disk to be created.
+            - If I(create_option=copy) then the value must be greater than or equal to the source's size.
     managed_by:
         description:
             - Name of an existing virtual machine with which the disk is or will be associated, this VM should be in the same resource group.
             - To detach a disk from a vm, explicitly set to ''.
             - If this option is unset, the value will not be changed.
-        version_added: 2.5
+        version_added: '2.5'
     attach_caching:
         description:
-            - "Disk caching policy controlled by VM. Will be used when attached to the VM defined by C(managed_by)."
-            - "If this option is different from the current caching policy,
-               the managed disk will be deattached from the VM and attached with current caching option again."
-            - "Allowed values: '', read_only, read_write."
+            - Disk caching policy controlled by VM. Will be used when attached to the VM defined by C(managed_by).
+            - If this option is different from the current caching policy, the managed disk will be deattached and attached with current caching option again.
         choices:
             - ''
             - read_only
             - read_write
-        version_added: 2.8
+        version_added: '2.8'
     tags:
         description:
             - Tags to assign to the managed disk.
+            - Format tags as 'key' or 'key:value'.
     zone:
         description:
-            - "Allowed values: 1, 2, 3, ''."
```

Comment:
In this line the values allowed were `1` or `2` or `3` or `.`. In the `choices` list below `.` is not listed, but `''` is. Can you confirm which list is correct?